main.cpp

```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    drummer
+//  action 1: drummer creates fills
+drummer.createFills();
+//  action 2: drummer kick the bass drum
+drummer.kickBassDrum();
+//  action 3:drummer hit the snare drum
+drummer.hitSnareDrum();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    pianist
+//  action 1: pianist plays notes
+pianist.playNotes();
+//  action 2: pianist plays chords
+pianist.playChords();
+//  action 3: pianist improvises
+pianist.improvise();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    guitarist
+//  action 1: guitarist sweeps
+guitarist.sweep();
+//  action 2: guitarist tunes
+guitarist.tune();
+//  action 3: guitarist tweak pedals
+guitarist.tweakPedals();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    bassist
+//  action 1: bassist plays root notes
+bassist.playRootNotes();
+//  action 2: bassist grooves
+bassist.groove();
+//  action 3: bassist slaps
+bassist.slap();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    singer
+//  action 1: singer harmonizes
+singer.harmonize();
+//  action 2: singer swings
+singer.swing();
+//  action 3: singer transposes
+singer.transpose();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    plant
+//  action 1: plant absorbs water
+plant.absorbWater();
+//  action 2: plant absorbs sunlight
+plant.absorbSunlight();
+//  action 3: plant releases oxygen
+plant.releaseOxygen();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: dog
+//  action 1: dog barks
+dog.bark();
+//  action 2: dog wags tail
+dog.wagTail();
+//  action 3: dog licks
+dog.lick();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    cat
+//  action 1: cat jumps
+cat.jump();
+//  action 2: cat purrs
+cat.purr();
+//  action 3: cat scratches
+cat.scratch();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    bird
+//  action 1: birf flies
+bird.fly();
+//  action 2: bird sings
+bird.sing();
+//  action 3: bird builds nest
+bird.buildNest();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.